File: src/main/java/com/projectswg/forwarder/resources/networking/packets/SeriousErrorAcknowledgeReply.java
package com.projectswg.forwarder.resources.networking.packets;

import java.nio.ByteBuffer;

public class SeriousErrorAcknowledgeReply extends Packet {
	
	public SeriousErrorAcknowledgeReply() {
		
	}
	
	public SeriousErrorAcknowledgeReply(ByteBuffer data) {
		decode(data);
	}
	
	public void decode(ByteBuffer data) {
		data.position(2);
	}
	
	public ByteBuffer encode() {
		ByteBuffer data = ByteBuffer.allocate(2);
		addNetShort(data, 0x1E);
		return data;
	}
	
}

File: src/main/java/com/projectswg/forwarder/resources/networking/packets/SeriousErrorAcknowledgeReply.kt
package com.projectswg.forwarder.resources.networking.packets

import com.projectswg.common.network.NetBuffer

class SeriousErrorAcknowledgeReply : Packet {
	
	constructor()
	constructor(data: NetBuffer) {
		decode(data)
	}
	
	override fun decode(data: NetBuffer) {
		data.position(2)
	}
	
	override fun encode(): NetBuffer {
		val data = NetBuffer.allocate(2)
		data.addNetShort(0x1E)
		return data
	}
	
}
